core/kt-osrd-sim-interlocking/src/test/kotlin/fr/sncf/osrd/sim/TestLocation.kt

```diff
@@ -19,10 +21,23 @@ class TestLocation {
         val infra = rawInfraFromRjs {
             // create a test switch
             val switchA =
-                movableElement("A", delay = 42L.milliseconds) {
-                    config("a", Pair(TrackNodePortId(0u), TrackNodePortId(1u)))
-                    config("b", Pair(TrackNodePortId(0u), TrackNodePortId(2u)))
-                }
+                node(
+                    "A",
+                    42L.milliseconds,
+                    StaticPool(),
+                    StaticPool(
```

Comment:
No port but non-empty config list with node port ids? That doesn't feel right, does it work properly? Or maybe it doesn't matter for the tests?

(the same applies to most test changes)

Reply:
I must admit I didn't dig deeper than "no call to [MovableElementDescriptorBuilder.port](https://github.com/osrd-project/osrd/blob/629612c8fa847ebc2c01226f0fb36830b5dc4766/core/kt-osrd-sim-infra/src/main/kotlin/fr/sncf/osrd/sim_infra/impl/RawInfraBuilder.kt#L79-L98) was done before" so I kept the same.
It looks like it doesn't matter for the tests, as there are not even track-sections on those tests.
So I went further on removing unused data after your remark in https://github.com/osrd-project/osrd/pull/6799/commits/c4d0a6a9f7a45fa3697f8a392413181cd6cc6e5d